package.json

```diff
@@ -17,13 +23,18 @@
   },
   "homepage": "https://github.com/mcollina/fastify-gql#readme",
   "devDependencies": {
+    "@types/graphql": "14.2.3",
```

Comment:
```suggestion
    "@types/graphql": "^14.2.3",
```

Reply:
This version must be fixed because this is the last version that contains type definitions. GraphQL v14.5.0 added Typescript definitions and with that addition the `@types/graphql` package became deprecated.

This dependency can be removed if the required `graphql` version is updated to `^14.5.0`